基本功能完善

tufu9441 opened the pull request.

tufu9441 committed:
主要功能完善

tufu9441 committed:
归档页面年份显示

tufu9441 committed:
fix bug

tufu9441 committed:
进一步完善

tufu9441 committed:
修正多说评论文章url的bug

icylogic commented:
谢谢你的帮助.

几个问题:
- [a8c0b4b](https://github.com/tufu9441/maupassant-hexo/commit/a8c0b4bd682be408b1dc36e5c671520d96f128d4)
  -  `+duoshuo: haomwei`, 这里填上了你自己的多说 id
  - 把 comments.jade 里多说的内容移到了 base 里, 这样 if theme.duoshuo 完全没有用了, 而且跟以后要增加的 disqus 冲突. 另外 base 是所有页面都会加载的, 而 comments 模块是只有 post 和 page 需要的. 
  - base.jade 加了一堆你自己的友链.
  - style.scss 加了一大串 base64 encode 过的代码, 不知道是什么.
  - title 应该利用我定义的 current_title 来改. PR 里的这个改法不能适应 tag/archives
  - search form 去掉了, 这个原主题有, 大部分 hexo 主题也有, 如果有问题可以修正, 我认为不应该去掉.

其他的我还没有看. 总之问题比较多, 希望你能在每个 commit 里尽量少改一些东西, 同时不要把自己私人的设置放到主题里. 如果你觉得某处可能使用户迷惑, 希望提供一个例子, 可以以注释的形式提供一个简单的 (比如 example.com) 说明.

tufu9441 commented:
@icylogic 我是把自己正在使用的版本直接上传了，所以难免会有我个人的信息和友情链接之类的，抱歉。

由于我在首页添加了一个侧边栏是显示最近评论的，所以需要多说的js代码在首页就需要加载。

style.scss里面的base64代码是右下角返回顶部按钮的图片和背景，这里直接使用的是data:image格式。

我之前是按照current title来改的，但是在任何一种情况下都没能生效，一直都是显示“config.title | config.title”的形式，不得已才出此下策……

search form是我个人不需要而去掉的，但我在使用过程中发现添加了搜索框以后，在windows的chrome下会搜索框最左边会出现一个蓝色的放大镜的小图标，显得不够美观，但在os x的chrome下不会出现，safari下会显示成一个不完全的痕迹，所以我把它去掉了……

非常感谢您的回复！

tufu9441 committed:
去除个人信息

tufu9441 commented:
我昨天已经按照您的建议，把包含私人设置的地方全都改掉了，并且在README.md里添加了对几处设置的说明。

关于归档以及分类标签页面的title我依然没有搞定，期待您在测试后能给出正确的方法。

另外，现在的归档页面里的显示顺序为：年份是按从早到晚的时间顺序排列，但每一年中的文章是按从晚到早的时间倒序排列的。我个人认为比较常见的归档形式最好是整体都是按时间倒序排列的，从最近期的文章开始排列，最早期的排在最后，即把目前年份的显示也改成从2015开始，然后是2014、2013……不知是否可行，谢谢！

tufu9441 committed:
添加代码高亮主题

tufu9441 committed:
修正BUG，增加disqus功能

tufu9441 commented:
我加上了disqus评论的模块，并且用一个很笨的方法实现了归档以及分类标签页面title的正确显示。~~非常期待您能抽空帮我看一下之前提到的归档页面显示顺序的问题~~，归档页面的显示顺序问题我又研究了一下也解决了，原来并不难哈哈……谢谢！

tufu9441 committed:
更改归档显示顺序为常见形式

tufu9441 committed:
完善侧边栏最近评论显示功能

tufu9441 committed:
增加Swiftype功能

tufu9441 committed:
添加标签云小工具

tufu9441 committed:
文章末尾添加显示分类信息

tufu9441 committed:
显示评论数

tufu9441 committed:
显示评论数CSS优化

tufu9441 committed:
修复Fancybox显示效果

tufu9441 committed:
增加tinysou站内搜索功能

tufu9441 committed:
bug fixed

tufu9441 committed:
bug fixed

tufu9441 committed:
duoshuo bug fixed

tufu9441 committed:
完善fancy box功能，并可以在部分图片上禁用fancybox

tufu9441 committed:
修复apple-touch-icon不显示的bug

tufu9441 committed:
更新README

tufu9441 committed:
添加Read More按钮

tufu9441 committed:
optimized

tufu9441 committed:
README updated

tufu9441 committed:
README updated

tufu9441 committed:
css optimized

tufu9441 committed:
README updated

v123582 committed:
fix languages/en.yml html tags

v123582 committed:
fix layout/post.jade condition

tufu9441 committed:
Merge pull request #1 from v123582/master

 fix some errors

tufu9441 committed:
优化英文界面

tufu9441 committed:
README updated

tufu9441 committed:
Update preview image

tufu9441 committed:
Footer information updated

tufu9441 committed:
文章后添加多说分享按钮

tufu9441 committed:
优化多说评论框样式，统一时间格式

tufu9441 committed:
优化文章中分类显示

tufu9441 committed:
A bug fixed and README updated

tufu9441 committed:
整合highlight css，使用hexo自带代码高亮功能

tufu9441 committed:
优化代码高亮样式

tufu9441 committed:
添加IcoMoon字体图标

tufu9441 committed:
README updated

jvalen committed:
Wrong file paths when the site is in a subdirectory bug fixed

css, js and font files had a wrong path when the site was put in a subdirectory,

e.g.

If the main hexo "_config.yml" file contains:

url: http://mysite.com/blog
root: /blog/

The path was something like:

  http://mysite.com/css/normalize.css

instead of:

  http://mysite.com/blog/css/normalize.css

tufu9441 committed:
Merge pull request #4 from jvalen/master

Wrong file paths when the site is in a subdirectory bug fixed

tufu9441 committed:
README updated

tufu9441 committed:
Post-navigator added, instead of page-navigator in posts.

tufu9441 committed:
a minor bug fixed

tufu9441 committed:
a font bug fixed

tufu9441 committed:
TOC function is supported now.

tufu9441 committed:
TOC optimized and README updated

tufu9441 committed:
README updated

tufu9441 committed:
A minor optimization.

tufu9441 committed:
bug fixed

tufu9441 committed:
Some minor optimizations.

tufu9441 committed:
Simplify the method of editing blogroll

tufu9441 committed:
improve disqus usage

tufu9441 committed:
README and LICENSE updated

tufu9441 committed:
README updated

tufu9441 committed:
a bug fixed

tufu9441 committed:
a bug fixed

tufu9441 committed:
fix a bug

Commit:
Modify the style.css and fix some bugs.

tufu9441 committed:
Merge pull request #6 from ArtinHuang/master

Modify the style.css and fix some bugs.

tufu9441 committed:
remove sharethis because of too many unnecessary js, and improve the disqus js.

tufu9441 committed:
Add other search engines and optimize search-form css

tufu9441 committed:
no message

tufu9441 committed:
no message

tufu9441 committed:
integrate menu settings to _config.yml

tufu9441 committed:
fix a bug

tufu9441 committed:
Optimization of fancybox and css

tufu9441 committed:
fancybox coding optimized

tufu9441 committed:
html coding optimized

tufu9441 committed:
A bug fixed

v123582 committed:
fix table invisible

tufu9441 committed:
post.excerpt bug fixed

v123582 committed:
fix code section error

v123582 committed:
Update style.scss

tufu9441 committed:
Merge pull request #9 from v123582/master

fix table invisible

tufu9441 committed:
table style optimized

tufu9441 committed:
minor improvements

tufu9441 committed:
A bug fixed

tufu9441 committed:
text align justify

tufu9441 committed:
fixed a bug in English display
